validator/sawtooth_validator/gossip/gossip_handlers.py

```diff
@@ -245,8 +247,15 @@ def handle(self, connection_id, message_content):
             # If we already have this block, don't forward it
             if not self._completer.get_block(obj.header_signature):
                 self._gossip.broadcast_block(obj, exclude, time_to_live=ttl)
+        elif tag == GossipMessage.CONSENSUS:
+            peer_message = ConsensusPeerMessage()
+            peer_message.ParseFromString(obj.message)
+
+            self._notifier.notify_peer_message(
+                message=peer_message,
+                sender_id=bytes.fromhex(connection_id))
```

Comment:
This needs to be the public key of the sending validator. I believe you can use this to get it: https://github.com/hyperledger/sawtooth-core/blob/master/validator/sawtooth_validator/gossip/gossip.py#L200